### DISCUSSION TITLE
Testing User Interaction with CasperWallet on DevNet/Testnet

### DISCUSSION THREAD

**Discussion by Marjan-sha** [casper-network/condor-info#49] [Q&A] [answered]
We are using CasperWallet for user interactions with our bridge . Will CasperWallet provide a version compatible with the new DevNet/(new)Testnet?
Additionally, we need to test the user flow, including how transactions are formed and sent. Will users be able to deploy their contracts to the DevNet/Testnet and use CasperWallet to interact with it before moving to production? (From Allbridge)

**Comment by melpadden** [comment]
@davidatwhiletrue

**Comment by davidatwhiletrue** [comment, marked as answer]
The current version of Casper Wallet can be used to sign `Deploy`s targeting the  `dev-net` network. Thus, any interaction with a contract that can be embedded in a `Deploy` object is possible already from a dApp.

Despite that, Casper Wallet will be updated to support Condor's new `TransactionV1` model and other features.
